tensorflow/lite/micro/kernels/softmax.cc

```diff
@@ -29,22 +29,39 @@ namespace micro {
 namespace activations {
 namespace {
 
+// Softmax parameter data that persists in user_data
+struct SoftmaxOpData {
+  struct SoftmaxParams params = {};
+  float table[256]{};
```

Comment:
Can you show me where this is used? I did a quick code search, and I don't see it being accessed.

Reply:
Done removed.
It was originally intended for LUT for int8 but was not used.